website/source/docs/configuration/interpolation.html.md

```diff
@@ -312,12 +312,13 @@ The supported built-in functions are:
   * `pathexpand(string)` - Returns a filepath string with `~` expanded to the home directory. Note:
     This will create a plan diff between two different hosts, unless the filepaths are the same.
 
-  * `pow(x, y)` - Returns the base `x` of exponential `y`.
+  * `pow(x, y)`  - Returns the base `x` of exponential `y` as a float.
 
     Example:
     * `${pow(3,2)}` = 9
     * `${pow(4,0)}` = 1
 
+
```

Comment:
ahh..my markdown renderer was asking to add one more line to have separation with the following point.

Removed extra line